more on Flutter web indexability (#10550)
flutter/website: src/content/platform-integration/web/faq.md

your landing page, marketing content, and help content—created using
search-engine optimized HTML.

> Having said that, as we mention in our [roadmap][], we plan to investigate
> search engine indexability of Flutter Web. To that end we've built a small
> website containing [Hawaii-themed space stories][space_hawaii], which we
> would like search engines to pick up and index.

Reviewer: **suggestion:** Would recommend tightening somewhat as the quantity of "we"s is a bit much.

```suggestion
That said, as mentioned in the [roadmap][], the Flutter team plans to investigate
search engine indexability of Flutter Web.
To that end, we built a small website containing [Hawaii-themed space stories][space_hawaii].
We intend for search engines to find and index this site.
```

Reviewer 2: Further feedback on Tony's feedback:

- It's "Flutter web" NOT "Flutter Web"
- "To that end, we built a small website containing [Hawaii-themed space stories][space_hawaii].
We intend for search engines to find and index this site."  =>
  To that end, we built a small website containing [Hawaii-themed space stories][space_hawaii],
 hoping that search engines find and index this site."

Author: Done. Thank you!
